status-im/status-react
# Enable custom bootnodes for nightly/releases

fixes #4587 
Fixes #4264 
fixes #664

### Summary:

This commit enable custom bootnodes for nightly & release.

### Review notes:

Only config change, but this will go through manual QA.

### Testing notes:

You should be able to add and use custom mailservers:

1) Go to profile
2) Advance
3) Bootnodes
4) Click on '+'
5) Enter name & address(without quotes): "enode://a8a97f126f5e3a340cb4db28a1187c325290ec08b2c9a6b1f19845ac86c46f9fac2ba13328822590fd3de3acb09cc38b5a05272e583a2365ad1fa67f66c55b34@167.99.210.203:30404"
6) Click on "use custom bootnodes"
7) You should be logged out
9) Log in
10) You should have general connectivity

- You should be able to edit custom bootnodes by clicking on it/editing/ click save. 
- You should be able to delete custom bootnodes by clicking on it & press Delete
- You should be able to scan a qr code to add a bootnode.

If custom bootnodes are  in use any edit/delete will log you out on successful save.

I could not test on IOS as I don't own a device, but some custom code had been written for it.

### Outstanding issues

Very difficult to tell whether it is using the new bootnode, I can verify that locally as I can check network traffic, but at the moment we don't have an isolated network to ensure it.

Depends on: https://github.com/status-im/status-go/pull/1003

status: ready
